src/components/LoanCards/AlgoliaLoanCardAdapter.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<div class="algolia-loan-card-adapterm column column-block">
+		<GridLoanCard
+			:key="loan.id"
+			:loan="adaptedLoan"
+			:is-visitor="true"
+			:items-in-basket="itemsInBasket" />
+	</div>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import GridLoanCard from '@/components/LoanCards/GridLoanCard';
+
+export default {
+	components: {
+		GridLoanCard
+	},
+	props: {
+		loan: {
+			type: Object,
+			default: () => {}
+		},
+		itemsInBasket: {
+			type: Array,
+			default: () => []
+		},
+	},
+	computed: {
+		adaptedLoan() {
+			// TODO: There must be a better way
```

Comment:
you could do each of these as a separate computed field

Reply:
agree. I'm a little concerned about what we're losing between this raw loan data the data we get back from graphql...so just hacked something together for now.